src/components/AdminPortal/EventRegistrantsModal/Modal/EventRegistrantsModal.spec.tsx

```
@@ -106,6 +110,80 @@ vi.mock('components/NotificationToast/NotificationToast', () => ({
+    }: InterfaceBaseModalProps) => {
+      if (!show) return null;
+
+      return (
+        <div data-testid={dataTestId || 'base-modal'}>
+          <div>
+            {title && <h2>{title}</h2>}
+            <button
+              aria-label="Close"
+              data-testid="modalCloseBtn"
+              onClick={onHide}
+            >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)